system-tests/ft-suite/src/test/kotlin/com/hotels/styx/admin/ProviderAdminInterfaceSpec.kt

```diff
@@ -119,6 +119,64 @@ class ProviderAdminInterfaceSpec : FeatureSpec() {
                 body shouldContain "/admin/providers/myMonitor/status"
                 body shouldContain "/admin/providers/mySecondMonitor/status"
             }
+
+            scenario("YAML configuration for all providers is available") {
+                val body = styxServer.adminRequest("/admin/providers/objects")
+                        .bodyAs(UTF_8)
+                body shouldContain """
+                    mySecondMonitor:
+                      type: "HealthCheckMonitor"
+                      tags: []
+                      config:
+                        objects: "bbb"
+                        path: "/healthCheck/y"
+                        timeoutMillis: 250
+                        intervalMillis: 500
+                        healthyThreshold: 3
+                        unhealthyThreshold: 2
+                    """.trimIndent()
+
+                body shouldContain """
+                    myMonitor:
+                      type: "HealthCheckMonitor"
+                      tags: []
+                      config:
+                        objects: "aaa"
+                        path: "/healthCheck/x"
+                        timeoutMillis: 250
+                        intervalMillis: 500
+                        healthyThreshold: 3
+                        unhealthyThreshold: 2
+                    """.trimIndent()
+
+                body shouldContain """
+                    originsFileLoader:
+                      type: "YamlFileConfigurationService"
+                      tags: []
+                      config:
+                        originsFile: "${originsFile.absolutePath}"
+                        ingressObject: "pathPrefixRouter"
+                        monitor: true
+                        pollInterval: "PT0.1S"
+                    """.trimIndent()
+            }
+
+            scenario("YAML configuration for a single provider is available") {
+                val body = styxServer.adminRequest("/admin/providers/objects/myMonitor")
```

Comment:
A small nag: IMHO the url should be `/admin/provider/objects` (The `provider` component should be in singular). `/admin/providers/objects` sounds wrong because both `providers` and `objects` are plural.

Do you think we could change it as a part of this PR, or would this warrant a separate PR?

Reply:
I'll change it as part of this PR, it's such a small change.